packages/core/src/components/header/header.tsx

```diff
@@ -52,17 +52,32 @@ export class Header {
   authStatus: "none" | "loggedIn" | "loggedOut" = "none";
 
   /**
-   * When the `authStatus` is `loggedOut` a loginUrl can be provided, the login button will render as an anchor.
+   * When the `authStatus` is `loggedOut` a loginUrl can be provided.
+   * The login button will then render as an anchor.
    */
   @Prop()
   loginUrl?: string;
 
   /**
    * The URL to open when the user activates "logout".
+   * If no URL is specified, a button element is used instead.
    */
   @Prop()
   logoutUrl?: string;
 
+  /**
+   * Show a help-button or link in the header
+   */
+  @Prop()
+  showHelp?: boolean;
```

Comment:
`show-help` is een boolean attribute, in dat geval is het fraaier om een default value `false` te zetten.

Een optional boolean property/attribute kan heeft de mogelijke values `undefined`, `false` en `true`. `undefined` en `false` hebben hetzelfde effect.

De documentatie die Stencil genereert wordt daarmee ook duidelijker. De default value komt daar ook te staan.